src/api/raw_transactions.js

```diff
@@ -0,0 +1,13 @@
+const { getInstance } = require('../qclient');
+
+const RawTransactions = {
```

Comment:
rename to `Util` class. I put the qtum functions based on the help menu of qtum client.